pkg/R/DataFrame.R

```diff
@@ -52,21 +53,108 @@ dataFrame <- function(sdf) {
 #' printSchema(df)
 #'}
 
-setGeneric("printSchema", function(df) { standardGeneric("printSchema") })
+setGeneric("printSchema", function(x) { standardGeneric("printSchema") })
 
 setMethod("printSchema",
-          signature(df = "DataFrame"),
-          function(df) {
-            sdf <- df@sdf
-            schemaString <- callJMethod(sdf, "printSchema")
+          signature(x = "DataFrame"),
+          function(x) {
+            schemaString <- callJMethod(x@sdf, "printSchema")
```

Comment:
When we test it locally, it works because the JVM and R process share the same console.

If we run those as separated process, then we can not see it in stdout of R.